backend/community/accounts/models.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
```

Comment:
Hmm. Did you create `User` model in place of django's User because you wanted to have less fields on it or for some other reason?

Reply:
I wanted a 'unique' constraint on email field.